webapp/packages/plugin-data-viewer/src/TableViewer/TableViewer.tsx

```diff
@@ -69,6 +71,10 @@ export const TableViewer = observer<TableViewerProps, HTMLDivElement>(
     const loading = dataModel?.isLoading() ?? true;
     const dataFormat = result?.dataFormat || ResultDataFormat.Resultset;
     const splitState = useSplitUserState('table-viewer');
+    const navRef = useListKeyboardNavigation(
+      '[data-presentation][tabindex]:not(:disabled), [data-presentation-tools][tabindex]:not(:disabled), [data-presentation-header][tabindex]:not(:disabled), [data-presentation-bar][tabindex]:not(:disabled)',
```

Comment:
just an idea (not required to implement right now): we should implement some kind of selector builder with chain of responsibility pattern cause now it is easy to miss-type something and get a bug